Makefile

```diff
@@ -5,9 +5,11 @@ GID:=$(shell id -g)
 
 ifeq ($(findstring .yelpcorp.com,$(shell hostname -f)), .yelpcorp.com)
 	export PIP_INDEX_URL ?= https://pypi.yelpcorp.com/simple
+	export NPM_CONFIG_REGISTRY ?= https://npm.yelpcorp.com/
```

Comment:
I've always seen this being set as lowercase `npm_config_registry`, but maybe it works regardless.

Reply:
You are right, they prefer the lower case but both are supposed to work according to the [docs](https://docs.npmjs.com/cli/v8/using-npm/config)
> Config values are case-insensitive, so NPM_CONFIG_FOO=bar will work the same. However, please note that inside scripts npm will set its own environment variables and Node will prefer those lowercase versions over any uppercase ones that you might set. For details see this issue.

However we're actually just hard-coding this later for `YARN_REGISTRY` or passing it on the commandline so we don't leave it to chance.